crates/fda/src/debug.rs

```diff
@@ -155,7 +161,267 @@ fn parse_metrics(file_name: &Path) -> anyhow::Result<()> {
     Ok(())
 }
 
-pub fn debug(action: DebugActions) {
+async fn unbundle_support_bundle(
```

Comment:
I would prefer this command to have a clearer name, since there are many things you can unbundle, including profiles.
Perhaps pipeline_from_bundle" would be clearer